roles/user-management/manage-atlassian-users/README.md

```diff
@@ -11,28 +11,34 @@ This is a sample of a YAML file containing the variables.
 
 ```
 ---
-atlassian_url: https://test.atlassian.net
-atlassian_username: admin
-atlassian_password: admin
-atlassian_users:
-  - firstname: Test
-    lastname: 123
-    password: password123
-    email: test@example.com
-    groups:
+atlassian:
+  url: https://example.atlassian.net
+  username: example
+  password: example
+
+  create:
+    user:
+      - firstname: Test
+        lastname: 123
+	password: example
+        email: test@example.com
+
+    add_user_to_groups:
```

Comment:
I wouldn't recommend just naming this `groups` and make it part of the user entry